robottelo/test.py

```diff
@@ -170,7 +171,10 @@ def setUpClass(cls):  # noqa
     def setUp(self):  # noqa
         """We do want a new browser instance for every test."""
         if self.driver_name.lower() == 'firefox':
-            self.browser = webdriver.Firefox()
+            self.browser = webdriver.Firefox(
+                firefox_binary=webdriver.firefox.firefox_binary.FirefoxBinary(
+                    self.driver_binary
```

Comment:
what happens if I dont specify `webdriver_binary` in properties file?  Will this line fail? If so, can you use a default value so it wont fail?